### ISSUE TITLE
ngx-golden-layout source

### ISSUE THREAD

**Issue by dpraul** [fresto32/ngx-golden-layout-2#2]
Hello! I'm glad you're working on this, but I wanted to let you know that ngx-golden-layout is actually open source! It is available here: https://github.com/EmbeddedEnterprises/ng6-golden-layout

The repo hasn't had much activity in a while, but the repo owner (@martin31821) has been busily contributing to the golden-layout core.

**Comment by martin31821** [comment]
Just as a heads up: I'm using the ngx package quite frequently for my own applications across several angular versions up to angular 12.
The reasoning why it hasn't been updated to golden-layout2 is that v2 lacks some features and differs in design which make a larger rewrite necessary, for which I don't have the time at the moment.

Contributions are welcome from my side. I'm acting as maintainer (merge & release process) along with @pbklink (most code work and review) to get a smooth v2 experience for the users. Again, contributions to everything are welcome. 

Mid-term, my goal would be to eventually move over the ngx-golden-layout to the golden-layout organization.
Hope that clarifies my intentions and reasons.

**Comment by taj-p** [comment]
Goodness gracious! I feel terrible - I did try looking for the source code - I think I got confused with the name changes. I've created an MR to remove those sentences from the README. 

@martin31821, thanks for chiming in. My colleagues and I are so grateful for the work you've done - everyone who sees it is amazed we can have such flexibililty in the web. That does make sense - it must be hard enough being the lead maintainer of the library itself.

I don't think I can commit to skilling up on the internals of golden layout 2, but I will try to get something more meaningful out for review into https://github.com/EmbeddedEnterprises/ng6-golden-layout in the long term. I've updated the readme to state that this is a short term solution until the official package is updated.
